_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -26649,6 +26665,13 @@
 /obj/item/reagent_containers/food/drinks/britcup{
 	desc = "Kingston's personal cup."
 	},
+/obj/item/storage/firstaid/brute{
+	pixel_x = 0
```

Comment:
dirty var

Reply:
To answer this question, This is what happens if you use the nudge-* commands in the map editor without enabling nudge as pixels.